src/main/resources/db/changelog/changes/0007-create-shopping-cart-table.yaml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping-cart-table
+      author: root
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:

```

Comment:
only one column?

Reply:
yes, I think it is enough for this model